src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -746,8 +753,16 @@ private void EnforceDoesNotReturn(SyntaxNode? syntaxOpt)
             DiagnosticBag diagnostics,
             bool createSnapshots,
             out SnapshotManager? snapshotManager,
-            ref ImmutableDictionary<Symbol, Symbol>? remappedSymbols)
+            ref ImmutableDictionary<Symbol, Symbol>? remappedSymbols,
+            bool avoidRewrite)
```

Comment:
The prefix `avoid` does not line up with the documentation that says "force". Feel like one of those needs to change.

Reply:
Splitting this up.

---
In reply to: [401064745](https://github.com/dotnet/roslyn/pull/42403#discussion_r401064745) [](ancestors = 401064745)